yjit_backend.c

```diff
@@ -4,8 +4,8 @@
 #include "yjit_backend.h"
 
 // List of scratch registers
-#define NUM_SCR_REGS 2
-#define SCR_REGS (ir_opnd_t[]){ IR_REG(RAX), IR_REG(RCX) }
+#define NUM_SCR_REGS 3
```

Comment:
Can we get away with just 2 scratch registers? That leaves more for allocation?

Reply:
Yup! I had to revisit the way I was handling `ir_mov` but it works with just 2 now.